src/interfaces/chain.h

```diff
@@ -279,9 +292,44 @@ class Chain
         virtual void chainStateFlushed(const CBlockLocator& locator) {}
     };
 
+    //! Options specifying which chain notifications are required.
+    struct NotifyOptions
+    {
+        //! Include undo data with block connected notifications.
+        bool connect_undo_data = false;
+        //! Include block data with block disconnected notifications.
+        bool disconnect_data = false;
```

Comment:
This field is only set by the coinstats index but never used.

Guessing that it is because the notifications now go through the `NotificationsProxy` class, who automatically fill-up the block data during disconnection.

While this is ok for the coinstats index, it is not that good for the other indexes. As they do not use the block data during disconnection, all these disk reads mean a resources waste for them.